### PR TITLE
Add generated HCNetSDK command-port captures

### PR DESCRIPTION
**Pull request by RenierM26**
## Summary

Adds native-Python HCNetSDK command-port support for generated multi-socket port-8000 capture flows, including fresh login, session-relative control-frame rendering, generated-plan JSON loading, and local stream save integration.

Key pieces:
- Generate command-port RSA/challenge login frames and parse reusable login/session state.
- Render reusable `0x63` control templates into native-style multi-socket command plans.
- Support generated-plan clip saves via CLI/client APIs, including `play_login_today` date refresh for native `0x111040` play-login templates.
- Preserve the native media-socket `IMKH` prefix behavior by default for `0x30000` media steps.
- Harden clear H.264 IDMX extraction to RTP payload type 96 and add startup-IDR skip/clean-IDR diagnostics.
- Add sanitized metadata, offline command-dump/H.264 summary helpers, and Frida diagnostics for native comparison.

## Practical validated recipe

For the current Husky command-port path, warning-free generated captures use:

- native media-prefix behavior on the media socket
- `body_tail_transform: "play_login_today"` for captured play-login templates
- RTP payload type 96 filtering for clear H.264 IDMX frames
- `--hcnetsdk-h264-skip-initial-idr-windows 1`

## Notes

This PR is now ready for review. Deeper PlayM4/decode callback tracing is intentionally kept in stacked draft PR #243 rather than expanding this supported generated-plan path.

## Verification

- `.venv/bin/python -m pytest -q`
- `.venv/bin/ruff check pyezvizapi tests`
- `.venv/bin/mypy pyezvizapi`
- `.venv/bin/pyright pyezvizapi`
- `node --check tools/apk-re/frida/ezviz-hcnetsdk-command-shape-capture.js`
- `node --check tools/apk-re/frida/ezviz-stream-transform-hook.js`
- `bash -n tools/apk-re/frida/run-ezviz-hcnetsdk-command-shape-hook`
- `git diff --check`

### CODE REVIEW

File: pyezvizapi/local_stream.py [hunk 1]
```diff
@@ -994,6 +1871,660 @@ def collect_local_stream_media_packets(
...
+            "unknown": 0,
+        },
+    }
+    samples = summary["samples"]
+    h264 = summary["h264"]
+    assert isinstance(samples, list)
+    assert isinstance(h264, dict)
+
+    for index, (start_code_offset, nal_start, end) in enumerate(
+        _h264_annexb_nal_spans(data),
```
[the remark this answers is not in the document]
Comment (RenierM26) [reply pyezvizapi/local_stream.py:1968]: Fixed in `2427d02` and still present in the latest head `9698d97`: `_h264_annexb_nal_spans()` now accepts both 3-byte and 4-byte Annex-B start codes, with a mixed-start-code regression test in `tests/test_local_stream.py`.

**Comment by RenierM26** [comment]
Follow-up state after marking ready:

- CI, Dependency Review, and CodeQL are green on `9698d97`.
- The Codex Annex-B start-code finding was fixed in `2427d02` with a regression test and its thread is resolved.
- Remaining CodeQL review comments were resolved as protocol-compatibility notes: the SHA-256, MD5 HMAC, and 1024-bit RSA operations are byte-for-byte HCNetSDK command-port compatibility requirements, not password-storage or general-purpose cryptography.

@codex review

File: pyezvizapi/local_stream.py [hunk 2]
```diff
@@ -768,11 +1570,35 @@ def copy_local_stream_to_mpegts(
...
+        if h264_trim_to_clean_idr_window:
+            annexb = trim_h264_annexb_to_first_clean_idr_window(
+                annexb,
+                ffmpeg_path=ffmpeg_path,
+                max_windows=h264_clean_idr_max_windows,
+            )
         process = _open_local_h264_mpegts_remux_process(ffmpeg_path)
         _copy_mpegps_payloads_to_mpegts([annexb], output, process=process)
         return
+    if h264_skip_initial_idr_windows or h264_trim_to_clean_idr_window:
```
[the remark this answers is not in the document]
Comment (RenierM26) [reply pyezvizapi/local_stream.py]: Fixed in `94cc5dd`: non-IDMX payloads now reject `h264_wait_for_clean_idr_window` alongside the other H.264 startup trim options, so MPEG-PS captures fail fast instead of silently extending the duration. Added regression coverage in `tests/test_local_stream.py`.

Verification:
- `.venv/bin/python -m pytest tests/test_local_stream.py -q`
- `.venv/bin/ruff check pyezvizapi/local_stream.py tests/test_local_stream.py`
- `.venv/bin/mypy pyezvizapi/local_stream.py`
- `.venv/bin/pyright pyezvizapi/local_stream.py`
- `git diff --check`

File: pyezvizapi/local_stream.py [hunk 3]
```diff
@@ -994,6 +1873,660 @@ def collect_local_stream_media_packets(
...
+        now = monotonic()
+        if clean_start_offset is None and now >= wait_deadline:
+            suffix = _h264_clean_idr_timeout_suffix(
+                first_decode_error=first_decode_error,
+                probe=last_probe,
+            )
+            raise PyEzvizError(
+                "Timed out waiting for a clean H.264 IDR window" + suffix
+            )
+        collected.append(packet)
```
[the remark this answers is not in the document]
Comment (RenierM26) [reply pyezvizapi/local_stream.py:2111]: Fixed in `06f2d3b`: after a clean IDR start is found, the capture deadline is now checked before appending the next packet, so a packet arriving at or after the requested clean-window deadline is excluded.

I also tightened the existing wait-mode timing test and added a direct regression for the deadline boundary in `tests/test_local_stream.py`.

Verification:
- `.venv/bin/python -m pytest tests/test_local_stream.py -q`
- `.venv/bin/ruff check pyezvizapi/local_stream.py tests/test_local_stream.py`
- `.venv/bin/mypy pyezvizapi/local_stream.py`
- `.venv/bin/pyright pyezvizapi/local_stream.py`
- `git diff --check`

**Comment by RenierM26** [comment]
Addressed the latest Codex finding in `06f2d3b` and resolved the review thread. Current checks on `06f2d3b` are green: CI, CodeQL, and Dependency Review.

@codex review
